grails-app/conf/application.yml

```diff
@@ -280,6 +280,8 @@ character:
     encoding: UTF-8
 
 http:
+    so:
```

Comment:
what's this for?

Reply:
SO_TIMEOUT is the timeout that a read() call will block.